osquery/tables/system/windows/hvci_status.cpp

```diff
@@ -0,0 +1,87 @@
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genHVCIStatus(QueryContext& context) {
+  Row r;
+  QueryData results;
+
+  const WmiRequest wmiSystemReq("SELECT * FROM Win32_DeviceGuard",
+                                (BSTR)L"ROOT\\MICROSOFT\\WINDOWS\\DEVICEGUARD");
+  const std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
+  if (wmiResults.empty()) {
+    LOG(WARNING) << "Error retreiving information from WMI.";
+    return results;
+  }
+  for (const auto& data : wmiResults) {
+    long vbsmethod;
+    long codepolicystatusmethod;
```

Comment:
Here and throughout, the variable declaration is pretty far away from it's use. It can be a little clearer if the variable is right next to it's use point, so for something like this you could do:
```
long code_policy_status;
data.GetLong("CodeIntegrityPolicyEnforcementStatus", code_policy_status);
// do something with code_policy_status
```

Reply:
Fixed.